whylog/config/log_type.py

```diff
@@ -14,3 +14,8 @@ def files_to_parse(self):
                 if file_source not in parsed_files:
                     parsed_files.add(file_source)
                     yield file_source
+
+    def does_belong_to_log_type(self, line_source):
```

Comment:
`log_type.does_belong_to_log_type` is not a good method name, please rename it appropriately. Maybe we can use `__contains__` and then `if line_source in log_type:`?